packages/amesos2/src/Amesos2_Superludist_def.hpp

```diff
@@ -319,6 +323,8 @@ namespace Amesos2 {
     if ( data_.options.SolveInitialized == SLUD::YES )
       function_map::SolveFinalize(&(data_.options), &(data_.solve_struct));
 
+    // gridexit of an older version frees SuperLU_MPI_DOUBLE_COMPLE,
```

Comment:
```suggestion
    // gridexit of an older version frees SuperLU_MPI_DOUBLE_COMPLEX,
```
I'm assuming a typo, unnecessary to fix here to avoid AT restart, just a very minor note for future cleanup